ir/instr.cpp

```diff
@@ -1324,6 +1346,38 @@ unique_ptr<Instr> Alloc::dup(const string &suffix) const {
 }
 
 
+vector<Value*> Malloc::operands() const {
+  return { size };
+}
+
+void Malloc::rauw(const Value &what, Value &with) {
+  RAUW(size);
+}
+
+void Malloc::print(std::ostream &os) const {
+  if (!getName().empty())
+    os << getName() << " = ";
+  os << "malloc " << *size;
+}
+
+StateValue Malloc::toSMT(State &s) const {
+  auto &[sz, np] = s[*size];
+  s.addUB(np);
+  // TODO: malloc's alignment is implementation defined.
+  return { s.getMemory().alloc(sz, 8, true), true };
```

Comment:
8 is not enough because it needs to return memory that is sufficiently aligned for all types. For a 64-bit machine it's usually 64-bit aligned.  I would just set it to 64 and ignore potential bugs related with over-assumptions regarding alignment (pls leave a comment though).